scripts/style.sh

```diff
@@ -163,6 +167,7 @@ s%^./%%
 '
 )
 
+echo $files
```

Comment:
intended?

Reply:
Thanks for spotting this..will remove this.